src/T2DMap.cpp

```diff
@@ -3079,6 +3082,54 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
         }
         connect(mapper, SIGNAL(mapped(QString)), this, SLOT(slot_userAction(QString)));
     }
+
+    updateSelectionWidget();
+    update();
+}
+
+void T2DMap::updateSelectionWidget() {
```

Comment:
Oh, very minor code styling point - the `{` should be on the next line.

Reply:
Corrected right away.